main.py

```
@@ -623,6 +626,51 @@ def _on_plugin_install_error(self, message: str, worker: _PluginInstallWorker):
+        confirm = QMessageBox.question(
+            self, "Uninstall Plugin",
+            f"Remove plugin '{choice}'? This cannot be undone.",
+            QMessageBox.StandardButton.Yes | QMessageBox.StandardButton.No,
+        )
+        if confirm != QMessageBox.StandardButton.Yes:
+            return
+
+        target = plugins_dir / choice
+        try:
+            shutil.rmtree(target)
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)